libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.h

```diff
@@ -94,6 +102,15 @@
 - (void)e:(nonnull Class)class message:(nonnull NSString *)message;
 
 /**
+ * Logs an error log line.
+ *
+ * @param class Class.
+ * @param format The format message, and optional arguments to expand in the format.
+ * @param ... The arguments to the message format string.
+ */
+- (void)e:(nonnull Class)class format:(nonnull NSString *)format, ...;
```

Comment:
These are just added for symmetry.  Seems like both method signatures should exist for each log level.